hazelcast/src/main/java/com/hazelcast/internal/metrics/renderers/CommaSeparatedKeyValueProbeRenderer.java

```diff
@@ -32,7 +32,7 @@
     @Override
     public void start() {
         sb.setLength(0);
-        sb.append("{").append("time").append("=").append(System.currentTimeMillis());
+        sb.append("").append("time").append("=").append(System.currentTimeMillis());
```

Comment:
This line could need some optimizing :)
`sb.append("time=").append(System.currentTimeMillis());`